src/librustdoc/html/static/css/settings.css

```diff
@@ -31,16 +24,33 @@
 
 .setting-line .radio-line input {
 	margin-right: 0.3em;
+	height: 1.2rem;
+	width: 1.2rem;
+	border: 1px solid;
+	outline: none;
+	-webkit-appearance: none;
+	cursor: pointer;
+	border-radius: 50%;
+}
+.setting-line .radio-line input + span {
+	padding-bottom: 1px;
+}
+
+.radio-line .setting-name {
+	width: 100%;
```

Comment:
Right now we have `.setting-line .radio-line` set to `flex` with `flex-wrap: wrap`, so the radio choices can wrap to the next line if the window is narrow. By making the heading 100% here you're making so that _always_ wraps, which will be confusing to the next person: "why put the flex there?"

So you should remove the `flex` and `flex-wrap` on `.setting-line .radio-line`. But keep it for `.setting-line .choices`. That will allow the choices to wrap if there are lots of them.

Reply:
Good point, removing them.